dispatch/static/manager/src/js/components/PollEditor/Poll.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react'
+import { Button, Intent } from '@blueprintjs/core'
+
+require('../../../styles/components/poll.scss')
+
+const COLOR_OPACITY = .8
+
+class Poll extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      showResults: true,
+    }
+  }
+
+  componentDidMount() {
+    this.update()
+  }
+
+  update() {
+    let answers = []
+    let votes = []
+    let pollQuestion = this.props.question ? this.props.question : 'Poll Question'
+    if(this.props.answers){
```

Comment:
Add a space after `if` as well (same as with `for`, and all control flow structures for that matter)